src/main/java/Duke.java

```diff
@@ -57,9 +57,40 @@ else if(userInput.startsWith("unmark")){
                 else {
                     tasks.get(index - 1).setUnmark();
                     System.out.println("-------------------------------------------------------------------------\n" + "OK, I've marked this task as not done yet:");
-                    System.out.println("[ ] " + tasks.get(index - 1).getDescription());
+                    System.out.println(tasks.get(index - 1).getDescription());
                 }
             }
+            else if(userInput.startsWith("todo")){
```

Comment:
Clean code, could think about refactoring the statements into functions to demonstrate SLAP.